Tests/ApolloCodegenTests/CodeGeneration/Templates/EnumTemplateTests.swift

```diff
@@ -129,57 +129,7 @@ class EnumTemplateTests: XCTestCase {
 
         // Escaped keywords
         ("associatedtype", nil, nil),
-        ("class", nil, nil),
```

Comment:
This is test cleanup - I'm not sure why we need all the escaped keywords in this test so I left two with different casing that require backticks.